.github/workflows/groovy-tests.yml

```diff
@@ -15,7 +15,3 @@ jobs:
       - name: Run Tests
         run: |
           ./gradlew test --info
-      - name: Upload Coverage Report
-        uses: codecov/codecov-action@v2
-        with:
-          files: ./build/reports/jacoco/test/jacocoTestReport.xml
```

Comment:
We need codecov for this as well right?

Reply:
No because in https://github.com/opensearch-project/opensearch-build/pull/3298 you removed jacoco.